### PR TITLE
RCSS cleanup

### PR DESCRIPTION
**Pull request by necessarily-equal**

### CODE REVIEW

File: ui/menu.rcss [hunk 1]
```diff
@@ -8,6 +8,19 @@ body.ingame {
 	background-color: rgba(0%,0%,0%,0%);
 }
 
+/* Block-style links */
+blocklink {
+	display: block;
+	color: #999999;
+	tab-index: auto;
+
```
Comment (DolceTriade) [inline ui/menu.rcss:16]: nit: extra line

Comment (necessarily-equal) [reply ui/menu.rcss]: fixed

**Comment by DolceTriade** [comment]
Sorry I'll try to test this later tonight

**Comment by DolceTriade** [comment]
ss

Alien health seems a little weird? I can't repro the same thing on 0.53.0/sync

**Comment by necessarily-equal** [comment]
Good catch @DolceTriade. Fixed.

**Comment by necessarily-equal** [comment]
Note that this also fixes the loading menu progress bar color

**Comment by necessarily-equal** [comment]
A screenshot for the loading screen:

image

**Comment by DolceTriade** [comment]
Will take a look later tonight

**Review by DolceTriade — APPROVED** [review]

**Comment by DolceTriade** [comment]
looks good now

**Pull request merged**